packages/next/client/future/image.tsx

```diff
@@ -645,6 +645,7 @@ const ImageElement = ({
         width={widthInt}
         height={heightInt}
         decoding="async"
+        fetchpriority={isLazy ? "low" : "high"}
```

Comment:
Should `fetchpriority` be assigned based on [`loading`](https://nextjs.org/docs/api-reference/next/image#loading) prop or based on [`priority`](https://nextjs.org/docs/api-reference/next/image#priority) prop?

Also, did you do any perf testing before and after?

Reply:
I think its best to wait on this before implementing since its Chrome only and its not clear how it will affect the existing `<link rel="preload">` we are using for `priority` prop (it sounds like this will eventually be a replacement once other browser support it)